app/models/manageiq/providers/vmware/cloud_manager/event_catcher/stream.rb

```diff
@@ -0,0 +1,105 @@
+require 'bunny'
+require 'thread'
+
+# Listens to RabbitMQ events
+class ManageIQ::Providers::Vmware::CloudManager::EventCatcher::Stream
+  def self.test_amqp_connection(options = {})
+    connection = nil
+    begin
+      connection = connect(options)
+      connection.start
+      return true
+    rescue Bunny::AuthenticationFailureError => e
+      $log.info("#{log_prefix} Failed testing rabbit amqp connection: #{e.message}")
```

Comment:
... and I think you can drop the `log_prefix` everywhere as well, since `_log` should do that automatically for you.